internal/controller/infrastructure/provisioner.go

```diff
@@ -38,6 +38,13 @@ type Provisioner struct {
 	log           logr.Logger
 }
 
+const startCommandTemplate = `(command -v systemctl > /dev/null 2>&1 && systemctl stop %s) || (command -v rc-service > /dev/null 2>&1 && rc-service %s stop) || (echo "Not a supported init system"; false)`
```

Comment:
Why do we need this, can't we rely on `k0s start` subcommand to handle this?

Reply:
It's basically a copy of [the command that we use to start k0s](https://github.com/k0sproject/k0smotron/blob/main/internal/controller/bootstrap/bootstrap_controller.go#L228). And of course, the name of the constant should be `stopCommandTemplate`. I've fixed that.